src/prefect/blocks/core.py

```diff
@@ -100,6 +100,37 @@ def _get_non_block_reference_definitions(object_definition: Dict, definitions: D
     return non_block_definitions
 
 
+def _is_subclass(cls, parent_cls) -> bool:
+    """
+    Checks if a given class is a subclass of another class.
```

Comment:
Let's note why we're creating a utility for this, like "Unlike `issubclass`, this will not throw an exception if `cls` is an instance instead of a type."